### PR TITLE
SameSite cookie default to Lax

### PR DESCRIPTION
**Pull request by taniwallach**
Change SameSite cookie setting to Lax as default, and add some explanations about the options and why Lax was selected as the default value.

### CODE REVIEW

File: conf/defaults.config [hunk 1]
```diff
@@ -918,7 +918,13 @@ $caliper{enabled} = 0;
 #         https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite
 #         https://tools.ietf.org/html/draft-west-cookie-incrementalism-00
 
-$CookieSameSite = "Strict";
+# Notes about the $CookieSameSite options:
+#   The "None" setting should only be used with HTTPS and when $CookieSecure = 1; is set below. The "None" setting is also las secure and can allow certain types of cross-site attacks.
+#   The "Strict" setting creak the links in the system generated fe
+edback emails when read in a web mail client.
+#   Due to those factors, the "Lax" setting is probably the optimal choice for typical WeBWorK servers.
```
Comment (drgrice1) [inline conf/defaults.config:925]: It looks like there are some typos that need to be fixed in both files.  `las secure`, `creak the links`?

There also seems to be a linebreak that shouldn't be (also in both files).

Comment (taniwallach) [reply conf/defaults.config:925]: Typing while exhausted and almost cross-eyed. Thanks for noticing. Force pushed the correction.

**Comment by drgrice1** [comment]
Looks good.

**Review by drgrice1 — APPROVED** [review]
This is ready to merge.

**Pull request merged**